src/pages/namsunkim/Feed/Feed.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { prefix } from '../config';
+import FeedIcon from './Components/FeedIcon/FeedIcon';
+import CommentList from './Components/CommentList/CommentList';
+import './Feed.scss';
+
+export default class Feed extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      feedArr: []
+    }
+  }
+
+  componentDidMount() {
+    const { api } = prefix;
+
+    fetch(`${api}/feedData.json`)
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          feedArr: data,
+        });
+      });
+  }
+
+  render() {
+    const { feedArr } = this.state;
+    const { feed, profile } = prefix;
+    return (
+      <>
+        {
+          feedArr.map(feedArr => {
```

Comment:
```suggestion
          feedArr.map(feed=> {
```

feedArr 안에 있는 요소들이 map돌면서 하나씩 할당될텐데 그걸 똑같이 feedArr라는 네이밍으로 짓는건 적절치 않습니다.
명확하게 변경해주시는게 좋아보이네요 예를들어 feed 등이 있습니다.

Reply:
넵!!